src/kv/cached.rs

```diff
@@ -56,15 +65,12 @@ impl CachedKvStore {
         let store = super::get_kv_store(nc, lattice_prefix, js_domain).await?;
         let linkdefs = Arc::new(RwLock::new(HashMap::new()));
         let claims = Arc::new(RwLock::new(ClaimsMap::default()));
-        let linkdefs_clone = linkdefs.clone();
-        let claims_clone = claims.clone();
+        let linkdefs_clone = Arc::clone(&linkdefs);
+        let claims_clone = Arc::clone(&claims);
```

Comment:
Did these end up being necessary?

Reply:
No. I can change them back if you want, but I also know that both ways are acceptable